ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml

```diff
@@ -3116,7 +3132,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1061</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.9</ns1:Doc_Sect>
-		<ns1:Description>[In t:ManagementRoleType Complex Type] The ManagementRoleType complex type specifies the roles that are associated with a user or application.&lt;76&gt;

```

Comment:
it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments